test/integration_tests/config.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 
-EXPECTED_TIME_TO_BOOT = 60  # seconds
+EXPECTED_TIME_TO_BOOT = 600  # seconds
```

Comment:
This is no a timeout. In the web server test, which will hopefully be reintroduced, this is the time we wait until we try to run `curl`. With this configuration it would wait for 10 minutes.

Reply:
If we need a timeout I suggest introducing a new variable. This is really about the **expected** time to boot.

Reply:
Ah, sorry, I changed the wrong value! Thanks!